sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/LogsHelperTests.cs

```diff
@@ -19,6 +19,55 @@ namespace Azure.Monitor.OpenTelemetry.Exporter.Tests
 {
     public class LogsHelperTests
     {
+        [Theory]
+        [InlineData(true)]
+        [InlineData(false)]
+        public void MessageIsSetToExceptionMessage(bool parseStateValues)
+        {
+            var logRecords = new List<LogRecord>();
+            using var loggerFactory = LoggerFactory.Create(builder =>
+            {
+                builder.AddOpenTelemetry(options =>
+                {
+                    options.IncludeFormattedMessage = true;
+                    options.ParseStateValues = parseStateValues;
```

Comment:
FYI - this is by default to `true` now. https://github.com/Azure/azure-sdk-for-net/pull/34433